vispy/visuals/tests/test_rectangle.py

```diff
@@ -7,9 +7,8 @@
 
 from vispy.scene import visuals, transforms
 from vispy.testing import (requires_application, TestingCanvas,
-                           run_tests_if_main)
+                           run_tests_if_main, assert_raises)
```

Comment:
eventually we should backport `raises` from `py.test`, but for now it was easier to change these two instances to use the `nose`-compatible syntax